egs/wsj/s5/steps/segmentation/detect_speech_activity.sh

```diff
@@ -56,7 +56,13 @@ acwt=0.3
 # e.g. --speech-in-sil-weight=0.0 --garbage-in-sil-weight=0.0 --sil-in-speech-weight=0.0 --garbage-in-speech-weight=0.3
 transform_probs_opts=""
 
+# Postprocessing options
 segment_padding=0.2   # Duration (in seconds) of padding added to segments 
+min_segment_dur=0   # Minimum duration (in seconds) required for a segment to be included
+                    # This is before any padding. This is an alternative to --min-speech-duration above.
```

Comment:
Please clarify that speech segments with length below this (before padding) will be discarded.  And for merge_consecutive_max_dur, you should probably clarify that this is after padding by --segment-padding seconds.